index.html

```diff
@@ -0,0 +1,1209 @@
+                      <p>Servers may accept different authentication protocols as well as credential verification methods.</p>
+                    </div>
+                  </div>
+
+                  <div class="note" id="origin-considerations" inlist="" rel="schema:hasPart" resource="#origin-considerations">
+                    <h3 property="schema:name"><span>Note</span>: Origin Considerations</h3>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>Typical Web browsers use origin based security. While it has many limitations, it is however ubiquitous.</p>
+                    </div>
+                  </div>
+
+                  <div class="issue" id="client-identification" rel="schema:hasPart" resource="#client-identification">
+                    <h3 property="schema:name"><span>Issue</span>: Client Identification</h3>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>Distinguishing social entities and clients: <a href="https://github.com/solid/web-access-control-spec/issues/81" rel="cito:citesAsRelated">issues/81</a>.</p>
+                    </div>
+                  </div>
+
+                  <p class="advisement">Source: <a href="https://github.com/solid/web-access-control-spec/issues/34" rel="cito:citesAsSourceDocument">issues/34</a>, <a href="https://github.com/solid/specification/issues/32" rel="cito:citesAsSourceDocument">issues/32</a>, <a href="https://github.com/solid/specification/issues/59">issues/59</a></p>
+                </div>
+              </section>
+            </div>
+          </section>
+
+          <section id="authorization-process" inlist="" rel="schema:hasPart" resource="#authorization-process">
+            <h2 property="schema:name">Authorization Process</h2>
+            <div datatype="rdf:HTML" property="schema:description">
+              <p>This section describes how implementations can determine the <a href="#effective-acl-resource" rel="cito:discusses">effective ACL resource</a> of a resource and hence which set of <a href="#authorization-conformance" rel="cito:discusses">conforming authorizations</a> to use.</p>
+
+              <p>Servers process access requests by <a href="#authorization-evaluation" rel="cito:discusses">evaluating the Authorizations</a> associated with referenced resources in order to determine whether the necessary access permissions are granted for a particular request.</p>
+
+              <section id="effective-acl-resource" inlist="" rel="schema:hasPart" resource="#effective-acl-resource">
+                <h3 property="schema:name">Effective ACL Resource</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>Servers enforce the effective ACL resource of a resource, whereas clients determine the effective ACL resource of a resource to perform read and write <a href="#authorization">Authorizations</a>.</p>
```

Comment:
Clients perform Authorizations? Did you mean clients perform operations?

Reply:
Good catch. I think I meant both operations and viewing/modifying Authorizations.

Updated to say:
>whereas clients determine the effective ACL resource of a resource to perform write operations.

That's based on https://github.com/solid/specification/issues/106#issuecomment-548399926

Reply:
Changed it to say "control operations" ( https://github.com/solid/web-access-control-spec/pull/83/commits/2fd1533ab1b672175f4781449b69a35ed9815b58 ) because client's reason to determine the #effective-acl-resource is not strictly for write operations. Client may want to only read the Authorizations or should even read existing Authorizations prior to writing.